test/test_ops.py

```diff
@@ -1770,5 +1770,53 @@ def test_is_leaf_node(self, dim, p, block_size, inplace):
         assert len(graph_node_names[0]) == 1 + op_obj.n_inputs
 
 
+class TestDiceLoss:
```

Comment:
I suggest to write this in a new file. Already `test_ops.py` is a huge file >1.5k lines of code.

Reply:
Aah quick check shows that all have written tests for losses to this file. Let this stay for now, thoughts @pmeier @NicolasHug ?